### DISCUSSION TITLE
[Question] When will the new version 14 be released?

### DISCUSSION THREAD

**Discussion by GuiSousa135** [LawnchairLauncher/lawnchair#4449] [Q&A]
The new Lawnchair 14 version is amazing, I know there are still some bugs and performance ups or crashes to be improved, but is there an estimate of when the official release for F-Droid will come out?

**Comment by SuperDragonXD** [comment]
Hello. Lawnchair 14 has been released [last April 3, 2024](https://github.com/LawnchairLauncher/lawnchair/releases/tag/v14.0.0-beta1). Regarding F-Droid, please see #1473.
